controllers/chat/sendSignedMessage.js

```diff
@@ -6,6 +6,7 @@ const Environment = require('../../config/environment');
 const {determineMessageType, processReplyMessage} = require('../../utils/chat');
 const UsersFunction = require('../../databases/functions/users');
 const {UserBlockedUser} = require('../../databases/models');
+const BetterSocialCore = require('../../services/bettersocial');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BetterSocialCore` module is imported but there's no check to ensure that it's properly initialized or ready to use. This could lead to runtime errors if the module fails to initialize correctly.
```diff
+ if (!BetterSocialCore.isInitialized()) {
+   throw new Error('BetterSocialCore is not initialized');
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->